webview/src/Api.ts

```diff
@@ -6,12 +6,15 @@ import { terminal } from 'virtual:terminal';
 
 // Use environment variable for API URL, fallback to relative URLs in production
 // Check window location to determine if we're in production
-const isProduction = window.location.hostname.includes('mentra.glass') || 
+const isProduction = window.location.hostname.includes('mentra.glass') ||
                     window.location.hostname.includes('onporter.run');
 
-const API_BASE_URL = isProduction 
-  ? 'https://translation-api.mentra.glass'
-  : (import.meta.env.VITE_API_URL || '');
+// Allow override via environment variable
+terminal.log('VITE_API_URL from env:', import.meta.env.VITE_API_URL);
+terminal.log('isProduction:', isProduction);
+
+const API_BASE_URL = import.meta.env.VITE_API_URL ||
+  (isProduction ? 'https://translation-aryan-api.mentra.glass' : '');
```

Comment:
should we change this back?
```suggestion
const API_BASE_URL = isProduction 
  ? 'https://translation-api.mentra.glass'
  : (import.meta.env.VITE_API_URL || '');
```

Reply:
Good call!